### DISCUSSION TITLE
Dependent’s API returning project properties

### DISCUSSION THREAD

**Discussion by RoshVenkatesh** [DependencyTrack/dependency-track#913] [Q&A]
Hi,
Is there a way to get the /dependency/component/{uuid} api to return project properties of dependent projects as well as a part of the payload?

**Comment by stevespringett** [comment]
The dependency API no longer exists in v4.0 and higher. That specific API is not capable of returning project properties in its response. The `/component/{uuid}` API in DT 4.0 doesn't return project properties either, but it would be simple to add support for it.
